tests/test_usb.py

```diff
@@ -22,6 +24,17 @@ def umockdev():
     return UMockdev.Testbed.new()
 
 
+def umockdev_has_working_remove():
```

Comment:
shouldn't we also try to enforce this in our CI images?

Reply:
We could but don't have to. At least with this PR we will automatically get the test in the future. As long as it's a single test I'm inclined to leave the CI as is.

Reply:
fair enough